pages/[league]/player/[id].tsx

```diff
@@ -46,6 +50,33 @@ const fetchPlayerName = (league: League, playerId: string) =>
   );
 
 export default ({ playerId, league }: { playerId: string; league: League }) => {
+  const router = useRouter();
+
+  const { portalView } = router.query;
+
+  const isIndexView = useMemo(
+    () => Boolean(portalView === undefined),
+    [portalView],
+  );
+  const { setColorMode } = useColorMode();
+
+  useEffect(() => {
+    if (
+      !isIndexView &&
+      portalView &&
+      (portalView === 'dark' || portalView === 'light')
+    ) {
+      if (portalView === 'dark') {
+        document.documentElement.classList.add('dark');
+        document.documentElement.classList.remove('light');
+      } else {
+        document.documentElement.classList.add('light');
+        document.documentElement.classList.remove('dark');
+      }
+      setColorMode(portalView);
+    }
+  }, [isIndexView, portalView, setColorMode]);
```

Comment:
Why don't we just do this instead:
```suggestion
  useEffect(() => {
    if (shouldShowIndexView) return;
    
    if (portalView === 'dark') {
        document.documentElement.classList.add('dark');
        document.documentElement.classList.remove('light');
    } else {
      document.documentElement.classList.add('light');
      document.documentElement.classList.remove('dark');
    }
    setColorMode(portalView);
  }, [shouldShowIndexView, portalView, setColorMode]);
```

If any issues occur lmk

Reply:
I would recommend moving this to when we actually implement the dark mode feature. As right now we won't be able to test whether or not it will work as intended